README.md

```diff
@@ -47,21 +47,22 @@ Known Limitations
 Code Structure
 --------------
 
-| File      | Content                                            |
-| ---------:|:---------------------------------------------------|
-| xpath.js  | the generated parser file.                         |
-| models.js | the underlying models files used in parsing.       |
-| src/      | contains the jison parser files.                   |
-| lib/      | external libraries that the parser depends on.     |
-| test/     | qunit tests for the parser and generator.          |
+| File           | Content                                            |
+| --------------:|:---------------------------------------------------|
```

Comment:
Guessing this alignment decision was intentional, although I find it hard to read once the markdown is converted to HTML. PR here https://github.com/dimagi/js-xpath/pull/26, but feedback is welcome.